libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/dh-processes-table/dh-processes-table.component.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CommonModule } from '@angular/common';
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  NgModule,
+  ViewChild,
+} from '@angular/core';
+import {
+  MatSort,
+  MatSortable,
+  MatSortModule,
+  Sort,
+} from '@angular/material/sort';
+import { MatTableModule } from '@angular/material/table';
+import { RouterModule } from '@angular/router';
+import { DhSharedUiDateTimeModule } from '@energinet-datahub/dh/shared/ui-date-time';
+
+import {
+  WattEmptyStateModule,
+  WattIconModule,
+  WattIconSize,
+} from '@energinet-datahub/watt';
+import { TranslocoModule } from '@ngneat/transloco';
+import { DhProcess } from '@energinet-datahub/dh/metering-point/data-access-api';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'dh-processes-table',
+  templateUrl: './dh-processes-table.component.html',
+  styleUrls: ['./dh-processes-table.component.scss'],
+})
+export class DhProcessesTableComponent implements AfterViewInit {
+  displayedColumns: string[] = [
+    'expand',
+    'name',
+    'createdDate',
+    'effectiveDate',
+    'status',
+    'hasErrors',
+  ];
+  iconSize = WattIconSize;
+  sortedData: DhProcess[] = [];
+  @Input()
+  processes?: DhProcess[];
```

Comment:
question: Would it be an issue to set the `processes` to an empty variable instead of being `undefined`?
```suggestion
  processes: DhProcess[] = [];
```

Reply:
The result should be the same from what I can see.